Client/src/app/app-routing.module.ts

```diff
@@ -1,15 +1,48 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
-import { LoginPageComponent } from './login-page/login-page.component'
-import { DashboardComponent } from './dashboard/dashboard.component'
-import { PageNotFoundComponent } from './page-not-found/page-not-found.component'
+import { LoginPageComponent } from './login-page/login-page.component';
+import { ContentWrapperComponent } from './content-wrapper/content-wrapper.component';
+import { PageNotFoundComponent } from './page-not-found/page-not-found.component';
+import { DashboardComponent } from './content-wrapper/dashboard/dashboard.component';
+import { LeavesComponent } from './content-wrapper/leaves/leaves.component';
+import { TimeentriesComponent } from './content-wrapper/timeentries/timeentries.component'
+import { AppComponent } from './app.component'
 
 
 const routes: Routes = [
-  { path: '', redirectTo: '/login', 'pathMatch': 'full' },
-  { path: 'pagenotfound', component: PageNotFoundComponent },
-  { path: 'login', component: LoginPageComponent },
-  { path: 'dashboard', component: DashboardComponent },
+  {
+    path: 'app',
```

Comment:
Login should be at route level, not within app

Reply:
Updated